packages/bvaughn-architecture-demo/components/Icon.tsx

```diff
@@ -62,7 +62,7 @@ export default function Icon({
       break;
     case "comment":
       path =
-        "M21.99 4c0-1.1-.89-2-1.99-2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h14l4 4-.01-18zM18 14H6v-2h12v2zm0-3H6V9h12v2zm0-3H6V6h12v2z";
+        "M7.87252 20.9906C8.05218 20.9906 8.21901 20.9496 8.37301 20.8674C8.53213 20.7853 8.71949 20.6493 8.93509 20.4593L11.8456 17.8645L16.6271 17.8722C17.4125 17.8722 18.0875 17.7208 18.6522 17.4179C19.2168 17.11 19.648 16.6736 19.9457 16.109C20.2486 15.5443 20.4 14.8719 20.4 14.0917V7.99346C20.4 7.21322 20.2486 6.54078 19.9457 5.97613C19.648 5.41148 19.2168 4.97773 18.6522 4.67488C18.0875 4.36689 17.4125 4.21289 16.6271 4.21289H6.97165C6.18628 4.21289 5.51127 4.36689 4.94662 4.67488C4.38711 4.97773 3.95592 5.41148 3.65306 5.97613C3.35021 6.54078 3.19878 7.21322 3.19878 7.99346V14.0917C3.19878 14.8719 3.35277 15.5443 3.66076 16.109C3.97389 16.6736 4.40251 17.1074 4.94662 17.4102C5.49587 17.7131 6.12725 17.8645 6.84076 17.8645H7.05635V20.0821C7.05635 20.3592 7.12821 20.58 7.27194 20.7442C7.4208 20.9085 7.621 20.9906 7.87252 20.9906ZM8.03422 11.0426C8.03422 10.7756 8.11891 10.5575 8.28831 10.3881C8.46284 10.2136 8.67843 10.1263 8.93509 10.1263H10.9062V8.15516C10.9062 7.8985 10.9909 7.68547 11.1603 7.51608C11.3297 7.34155 11.5453 7.25429 11.8071 7.25429C12.0689 7.25429 12.2845 7.34155 12.4539 7.51608C12.6284 7.68547 12.7157 7.8985 12.7157 8.15516V10.1263H14.6945C14.9512 10.1263 15.1642 10.2136 15.3336 10.3881C15.503 10.5575 15.5877 10.7756 15.5877 11.0426C15.5877 11.2992 15.503 11.5148 15.3336 11.6893C15.1642 11.8587 14.9512 11.9434 14.6945 11.9434H12.7157V13.9146C12.7157 14.1712 12.6284 14.3842 12.4539 14.5536C12.2845 14.723 12.0689 14.8077 11.8071 14.8077C11.5453 14.8077 11.3297 14.723 11.1603 14.5536C10.9909 14.3842 10.9062 14.1712 10.9062 13.9146V11.9434H8.93509C8.67843 11.9434 8.46284 11.8587 8.28831 11.6893C8.11891 11.5148 8.03422 11.2992 8.03422 11.0426Z";
```

Comment:
Ooo, I'm curious where this SVG came from. It doesn't look like it quite fits the 24x24 view box as tightly as the others:
![Screen Shot 2022-08-10 at 10 34 22 AM](https://user-images.githubusercontent.com/29597/183928778-16863636-6267-46ee-8099-267711fcb297.png)

Reply:
This is our current add comment icon. I tightened it up a little bit on the bounds, but I'll be going through all of our icons after this ships so it will be changing here soon. I want to normalize all of our icons in design and dev and get things down to one component and system.

Reply:
Sure, sure. No problem with the shape of the icon at all. My concern was that it doesn't "fit" the view box very well, which means it's going to look odd (smaller) relative to the other icons